src/components/experimental/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import styled from 'styled-components';
+import { ButtonProps, Button } from 'react-aria-components';
+import { IconProps } from '../../../icons';
+import { getSemanticValue } from '../../../essentials/experimental';
+
+export interface IconButtonProps extends ButtonProps {
+    isActive?: boolean;
+    variant?: 'standard' | 'tonal';
+    Icon: React.FC<IconProps>;
+    onPress: () => void;
+}
+
+const StandardIconContainer = styled(Button)<Omit<IconButtonProps, 'Icon'>>`
+    height: 5rem;
+    width: 5rem;
+    border-radius: 100%;
+    background-color: transparent;
+    border-color: transparent;
+
+    /* we create a before pseudo element to mess with the opacity (see the hovered state) */
+    &::before {
+        position: absolute;
+        content: '';
+        border-radius: inherit;
+        opacity: 0;
+        height: 5rem;
+        width: 5rem;
+    }
+
+    /* we want to change the opacity here but not affect the icon, so we have to use the before pseudo element */
+    &[data-hovered]::before {
+        opacity: 0.16;
+        background-color: ${getSemanticValue('on-surface')};
+    }
+
+    /* this is to position the svg relative to the before pseudo element */
+    > svg {
+        padding: 1rem;
+        width: 3rem;
+        height: 3rem;
+    }
+
+    &:not([data-disabled]) {
+        color: ${props => (props.isActive ? getSemanticValue('interactive') : getSemanticValue('on-surface'))};
+    }
+
+    &[data-disabled] {
+        opacity: 0.38;
+    }
+`;
+
+const TonalIconContainer = styled(Button)<Omit<IconButtonProps, 'Icon'>>`
+    height: 7rem;
```

Comment:
By default 1rem = 16px, so it should be width & height = 3.5rem

Reply:
Thanks I had a feeling this was doubly large but I was like ok figma says so 😬 

Will halve this!!

Reply:
Done! ✅